### ISSUE TITLE
Allow copy pasting text/uri-lists from the explorer 

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#181587]
This is the same as #181373 but for copy paste instead of drag and drop

**Repo**
1. On desktop
1. Copy file from VS Code explorer
2. Try pasting it into the chrome address bar

**Expected**
We should paste a `file://` uri

**actual**
Nothing is pasted

**Comment by mjbvz** [comment]
We should also write `text/plain` in this case. For this, you can try pasting into a normal text editor instead of the chrome address bar

**Comment by mjbvz** [comment]
Actually already tracked by #91132 and blocked by https://github.com/electron/electron/issues/11838
